neutrons/addie
# Fixes #215 - Correct case on shape names for Mantid

Fixes #215 

*To test*
----------
Full reduction: Load `tests/nano3_test_table.json` into the table and run reduction. Ensure it does not raise an error related to shape names reported in Issue #215 

Quick: Add a print statement for shape at `addie/processing/mantid/master_table/master_table_exporter.py` after [line 178](https://github.com/neutrons/addie/blob/c4ae69b12d0370ecefd5f06dd022a6cd31ad5b07/addie/processing/mantid/master_table/master_table_exporter.py#L178). Ensure it prints out correct cased shape value.
